fv3core/stencils/riem_solver3.py

```diff
@@ -129,9 +129,9 @@ def compute(
     shape = w.shape
     domain = (grid.nic, grid.njc, km + 2)
     riemorigin = (grid.is_, grid.js, 0)
-    dm = copy(delp)
-    cp3 = copy(cappa)
-    pe_init = copy(pe)
+    dm = copy(delp, origin=grid.default_origin())
+    cp3 = copy(cappa, origin=grid.default_origin())
+    pe_init = copy(pe, origin=grid.default_origin())
```

Comment:
default_origin is (0, 0, 0) and since you've modified the copy method to choose this if origin is not specified, you don't need this change. But, it doesn't hurt either.

Reply:
Good point. I'll remove those!